abigen failed with latest dev version

#### System information

abigen version: 
```shell 
abigen version 2.31.0-dev
```

OS & Version: Windows & MacOS

Commit hash : 
```shell
abigen --abi=./abis/erc20/erc20.abi --pkg=erc20 --out=./abis/erc20/erc20.go
```

#### Expected behaviour
normal output an erc20.go file


#### Actual behaviour

```shell
panic: invalid argument, not cli.Flag type: cli.StringFlag

goroutine 1 [running]:
github.com/ledgerwatch/erigon/cmd/utils.CheckExclusive(0x12f0a60?, {0xc000493900, 0x4, 0xfffffffe00000000?})
        G:/github_repo/erigon/cmd/utils/flags.go:1387 +0x55b
main.abigen(0xc000403b80?)
        G:/github_repo/erigon/cmd/abigen/main.go:122 +0x10e
github.com/urfave/cli/v2.(*Command).Run(0xc000403b80, 0xc0016c3400, {0xc000088040, 0x4, 0x4})
        C:/Users/jiang/go/pkg/mod/github.com/urfave/cli/v2@v2.23.5/command.go:271 +0xaab
github.com/urfave/cli/v2.(*App).RunContext(0xc0001f8f00, {0x1979cc0?, 0xc000042110}, {0xc000088040, 0x4, 0x4})
        C:/Users/jiang/go/pkg/mod/github.com/urfave/cli/v2@v2.23.5/app.go:329 +0x665
github.com/urfave/cli/v2.(*App).Run(...)
        C:/Users/jiang/go/pkg/mod/github.com/urfave/cli/v2@v2.23.5/app.go:306
main.main()
        G:/github_repo/erigon/cmd/abigen/main.go:272 +0x7b
```


#### Steps to reproduce the behaviour


#### Backtrace

````
[backtrace]
````

--- a/cmd/abigen/main.go
+++ b/cmd/abigen/main.go
@@ -119,7 +119,7 @@ func init() {
 }
 
 func abigen(c *cli.Context) error {
-	utils.CheckExclusive(c, abiFlag, jsonFlag, solFlag, vyFlag) // Only one source can be selected.
+	utils.CheckExclusive(c, &abiFlag, &jsonFlag, &solFlag, &vyFlag) // Only one source can be selected.
 	if c.String(pkgFlag.Name) == "" {
 		utils.Fatalf("No destination package specified (--pkg)")
 	}

